README.md

```diff
@@ -0,0 +1,27 @@
+# Sern CLI
+
+Our CLI is allowing you to create & streamline your Discord bot project without writing a single line of code!
```

Comment:
Our CLI allows you to setup and manage Discord bot projects without writing a line of code!

Reply:
Ok